<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlideshowLayerDialog</class>
 <widget class="QWidget" name="SlideshowLayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Slideshow layer settings</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="StyledFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="filenameLbl">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Image files:</string>
        </property>
        <property name="buddy">
         <cstring>listWidget</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="moveUpBtn">
        <property name="text">
         <string>Move image up</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="7">
       <widget class="QListWidget" name="listWidget"/>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="removeBtn">
        <property name="text">
         <string>Remove image</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="addBtn">
        <property name="text">
         <string>Add new image</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="moveDownBtn">
        <property name="text">
         <string>Move image down</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="StyledFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Time between images:</string>
        </property>
        <property name="buddy">
         <cstring>delayEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Transition duration:</string>
        </property>
        <property name="buddy">
         <cstring>transitionEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="UnitLineEdit" name="delayEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="UnitLineEdit" name="transitionEdit"/>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="seqBtn">
           <property name="text">
            <string>Sequential</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="randomBtn">
           <property name="text">
            <string>Random</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Display order:</string>
        </property>
        <property name="buddy">
         <cstring>seqBtn</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Transition style:</string>
        </property>
        <property name="buddy">
         <cstring>inThenOutBtn</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="inThenOutBtn">
           <property name="text">
            <string>Fade in, then out</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="inAndOutBtn">
           <property name="text">
            <string>Fade in and out</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StyledFrame</class>
   <extends>QFrame</extends>
   <header>Widgets/styledframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UnitLineEdit</class>
   <extends>QLineEdit</extends>
   <header>Widgets/unitlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>addBtn</tabstop>
  <tabstop>removeBtn</tabstop>
  <tabstop>moveUpBtn</tabstop>
  <tabstop>moveDownBtn</tabstop>
  <tabstop>delayEdit</tabstop>
  <tabstop>transitionEdit</tabstop>
  <tabstop>inThenOutBtn</tabstop>
  <tabstop>inAndOutBtn</tabstop>
  <tabstop>seqBtn</tabstop>
  <tabstop>randomBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
